scripts/upgrade_forkid.sh

```diff
@@ -0,0 +1,259 @@
+# rollup 11 -> 13
+# rollup 12 -> 13
+# rollup  9 -> 11
+# rollup  9 -> 12
+# rollup  9 -> 13
+
+# Check we receive 2 params
+if [ "$#" -ne 2 ]; then
+    echo "Usage: upgrade_forkid.sh <source_forkid> <target_forkid>"
+    exit 1
+elif ! [[ $1 =~ ^[0-9]+$ ]] || ! [[ $2 =~ ^[0-9]+$ ]]; then
+    echo "Forkids must be integers"
+    exit 1
+# check source forkid is 9, 11 or 12
+elif [ $1 -ne 9 ] && [ $1 -ne 11 ] && [ $1 -ne 12 ]; then
+    echo "Source forkid must be 9, 11 or 12"
+    exit 1
+# check target forkid is 11, 12 or 13
+elif [ $2 -ne 11 ] && [ $2 -ne 12 ] && [ $2 -ne 13 ]; then
+    echo "Target forkid must be 11, 12 or 13"
+    exit 1
+# check target forkid is greater than source forkid
+elif [ $1 -ge $2 ]; then
+    echo "Target forkid must be greater than source forkid"
+    exit 1
+fi
+
+SOURCE_FORKID=$1
+TARGET_FORKID=$2
+ERIGON_IMAGE=hermeznetwork/cdk-erigon:v2.61.4-RC1
+STACK_NAME=upgradeCDK-$(tr -dc A-Za-z0-9 </dev/urandom | head -c 13; echo)
+
+if [ $TARGET_FORKID -eq 11 ]; then
+    TAG_TARGET_FORKID=v7.0.0-fork.10-fork.11
+    UPGRADED_PARAMS_FILE=.github/tests/combinations/fork11-new-cdk-stack-rollup.yml
+elif [ $TARGET_FORKID -eq 12 ]; then
+    TAG_TARGET_FORKID=v8.0.0-rc.4-fork.12
+    UPGRADED_PARAMS_FILE=.github/tests/combinations/fork12-new-cdk-stack-rollup.yml
+elif [ $TARGET_FORKID -eq 13 ]; then
+    TAG_TARGET_FORKID=v8.1.0-rc.2-fork.13
+    UPGRADED_PARAMS_FILE=.github/tests/combinations/fork13-new-cdk-stack-rollup.yml
+fi
+
+if [ $SOURCE_FORKID -eq 9 ]; then
+    PARAMS_FILE=.github/tests/combinations/fork9-new-cdk-stack-rollup.yml
+elif [ $SOURCE_FORKID -eq 11 ]; then
+    PARAMS_FILE=.github/tests/combinations/fork11-new-cdk-stack-rollup.yml
+elif [ $SOURCE_FORKID -eq 12 ]; then
+    PARAMS_FILE=.github/tests/combinations/fork12-new-cdk-stack-rollup.yml
+fi
+
+KURTOSIS_CONFIG=upgrade_from_${SOURCE_FORKID}_to_${TARGET_FORKID}_enclave_${STACK_NAME}.json
+cp $PARAMS_FILE $KURTOSIS_CONFIG
+sed -ni '/cdk_erigon_node_image/!p' $KURTOSIS_CONFIG
+echo "  cdk_erigon_node_image: $ERIGON_IMAGE" >> $KURTOSIS_CONFIG
+
+# DEPLOY STACK
+kurtosis run --enclave $STACK_NAME --args-file $KURTOSIS_CONFIG .
+
+# SERVICE NAMES
+SVC_SEQUENCER=cdk-erigon-sequencer-001
+SVC_RPC=cdk-erigon-rpc-001
+SVC_CONTRACTS=contracts-001
+SVC_SLESS_EXECUTOR=zkevm-stateless-executor-001
+SVC_BRIDGE=zkevm-bridge-service-001
+SVC_PROVER=zkevm-prover-001
+SVC_CDKNODE=cdk-node-001
+
+# send test tx
+PRIV_KEY=0x12d7de8621a77640c9241b2595ba78ce443d05e94090365ab3bb5e19df82c625
+cast send --rpc-url $(kurtosis port print $STACK_NAME $SVC_RPC rpc) --legacy --private-key $PRIV_KEY --value 0.01ether 0x0000000000000000000000000000000000000000
+
+# Halt sequencer
+echo "Halting sequencer..."
+kurtosis service exec $STACK_NAME $SVC_SEQUENCER \
+    "HALTON=\$(printf \"%d\\n\" \$((\$(curl -s -X POST -H \"Content-Type: application/json\" -d '{\"method\":\"zkevm_batchNumber\",\"id\":1,\"jsonrpc\":\"2.0\"}' http://localhost:8123 | jq -r .result)+2))); \
```

Comment:
Not sure but maybe it would be easier to just use:

``` bash
cast rpc --rpc-url http://localhost:8123 zkevm_batchNumber
```

Reply:
the issue for this is that Erigon image does not have cast